lms/resources/_js_config/__init__.py

```diff
@@ -314,12 +314,23 @@ def enable_instructor_toolbar(self, enable_editing=True, enable_grading=False):
                     username=grading_info.h_username,
                     display_name=grading_info.h_display_name,
                 )
+
+                lis_result_sourced_id = grading_info.lis_result_sourcedid
+                if self._application_instance.lti_version == "1.3.0":
```

Comment:
Will this also apply to LTI 1.3.1 and so on if they ever come out?

Reply:
Yeah, most likely. 

We'd probably move this to a enum instead of looking at the raw string.